terms.html

```diff
@@ -169,8 +169,12 @@
 <a>verifiable presentation</a> for processing. Other specifications might refer
 to this concept as a <dfn data-lt="relying parties">relying party</dfn>.
   </dd>
-  <dt><dfn data-lt="URI|URIs">URI</dfn></dt>
+  <dt><dfn data-lt="URL|URLs">URL</dfn></dt>
   <dd>
-A Uniform Resource Identifier, as defined by [[RFC3986]].
+A Uniform Resource Locator, as defined by [[URL]]. This specification does
+not use the term URI or IRI because those terms have been deemed to be confusing
+to Web developers. This specification uses the phrase "nondereferenceable URL"
+when referring to the concept of a URI or IRI, as defined in [[?RFC3986]] and
+[[?RFC3987]], respectively.
```

Comment:
I don't think "non-deferenceable URL" is very helpful.

Even URNs are not "non-dereferenceable". Every URI "references" something, since the point of a URI is to identify  (reference) a logical or physical resource. But not every URI is a "locator". That means you can't always send a message to a commonly resolvable place specified within the URI using a protocol specified within the URI to "locate" the referenced resource. Maybe none of that is all that useful either and people just get lost in the details.

I do think it's more likely to be helpful to say whether there's an expectation that a URI (now just "URL") "resolves" to something (and what that something is expected to be) where necessary. Otherwise, all that matters is that we say "this is where you put an identifier of a specific (and standard) format, i.e., a "URL", to reference something". If we aren't restricting "what you can do / should be able to with that identifier" then it doesn't really matter and we don't need to say anything about "locators", etc.

Reply:
> I don't think "non-deferenceable URL" is very helpful.

Ok, I'll make another pass on this PR with that in mind. The  "non-deferenceable URL" is not popular. :)

Reply:
Fixed in eb0f404d62f99fa01c35af74072be5869996596f.

I went down a rabbit hole trying to see if we should pull the [WHATWG Fetch spec](https://fetch.spec.whatwg.org/#url) in before realizing that if we did that, we would definitely step on the landmine that @OR13 was concerned about stepping on -- turns out WHATWG Fetch does not define "fetching" for anything other than a handful of schemes (http(s), ws(s), data, file, etc.). So, I kept it generic by saying "dereference" and DID NOT use the term "fetch", and said that "how you dereference is covered by the URL scheme specification".